packages/cli/src/lib/commands/collect/utils/persist/persist-flow.unit.test.ts

```diff
@@ -0,0 +1,124 @@
+import { join } from 'node:path';
+import * as fs from 'node:fs';
+import * as utils from './utils';
+import * as reportUtils from '../report/utils';
+import * as mdReportUtils from '../../../assert/utils/md-report';
+import { persistFlow } from './persist-flow';
+import { writeFile } from '../../../../core/file';
+import { log } from '../../../../core/loggin';
+import { ReducedReport } from '../report/types';
+
+import { UserFlow } from '../../../../hacky-things/lighthouse';
+
+jest.mock('node:fs');
+jest.mock('../../../../hacky-things/lighthouse')
+jest.mock('../../../../core/file');
+jest.mock('../../../../core/loggin');
+jest.mock('./utils');
+jest.mock('../../../assert/utils/md-report');
+jest.mock('../report/utils');
```

Comment:
If you provide a simple report name, you can then match the whole returned result name array value in the tests.
```suggestion
jest.mock('../report/utils', () => ({
  createReducedReport: jest.fn(),
  toReportName: jest.fn().mockReturnValue('report'),
}));
```